project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md

```diff
@@ -28,20 +29,21 @@ Trying a new format today to keep it more
 - connections turing course: late june-early july (following convo)
 
 **MS:** thanks for the timeline and overview.
+
 # Next week
 - More outreach to individuals. - MS: 💪🏽 
 - Compile upcoming conferences to post on PR + start asking people if they'd like to rep TTW :) (would be super excited if possible!) - MS: we have generally aimed for at least 50% talks by community members. Keeping that or increasing that will be a good target. Do take some time to look at the previous reports and promotion pack to think about what can be updated.
 - Had a couple of ideas for engagement with TTW (wondering about the timeline for implementing them - can I start experimenting with one of these next week more concretely?):
   - Turing Way coffee hours. - **MS:** Sounds great! 🎉  Maybe something to think about when your 1:1 meetings start to reduce in number.
   - Turing Way reading group (read 1 section a week?) - **MS:** KW and I were discussing this with AR to co-lead this with you. Can you discuss this with AR and involve him in designing your plans in May?
   - Slack automated welcoming message? (would love to co-create this, maybe in the community share out?) - **MS**: sent you some info on Slack for configuring Greetbot.
-  - Github Issue templates  - **MS**: This will be fantastic! Maybe a low hanging fruit to lead on get some interested folkd invlolved. You can proposee this for the Book Dash collab idea too.
+  - Github Issue templates  - **MS**: This will be fantastic! Maybe a low hanging fruit to lead on and get some interested folks invlolved. You can proposee this for the Book Dash collab idea too.
 
 # Long term ideas
 - Review onboarding process - **MS**: Sounds fantastic. We should definitely create a process to be shared in the Community Handbook. So far it has happened through Coworking, Collab Cafe and workshops. As you know, I have used this doc in the past for onboarding calls: https://hackmd.io/@turingway/demo-intro (which is an overview of the GitHub and book). We can get people involved in understanding what was helpful and what be improved -- some of these can go to a chapter.
 - Shared slack channels with other open science / related open source projects? (I've seen this happen on OSM) - **MS:** This is a Slack pro feature. We may not have funding for the pro account (as Turing institute has already a fully paid account and they will likely not approve extra workspace request) - but we can explore this with ABL if you can assess the pros and cons carefully.
 - UX/UI design sprint for section naming & organisation
-(just to keep my pie in the sky brain awake!) - **MS**: not sure what this means, happy to discuss! We probably need to have a UX/UI plan for the entire project (as well as merge the recent UX work by REG team.
+(just to keep my pie in the sky brain awake!) - **MS**: not sure what this means, happy to discuss! We probably need to have a UX/UI plan for the entire project (as well as merge the recent UX work by REG team).
```

Comment:
The only comment here, but I was thinking of the meeting with KW & you, about agenda-less meetings and the need to leave room for big ideas. Hence, this little section on a few ideas that have come to mind over the past month & after conversations with people :)

Reply:
Good idea! I like this section very much.